### DISCUSSION TITLE
Planned support for anonymous auth?

### DISCUSSION THREAD

**Discussion by gaetandezeiraud** [go-pkgz/auth#195] [Q&A] [answered]
Planned support for anonymous auth? Can be useful to authenticate a user with a device id and allow it to attach a "real" way to authenticate later.

**Comment by umputun** [comment, marked as answer]
Anonymous auth is already possible with the direct provided, this is [how it is done](https://github.com/umputun/remark42/blob/master/backend/app/cmd/server.go#L982)  in one of the projects using auth

_pls, use Discussions for such types of questions. Issue is for the real issues/bugs only_

**Reply by gaetandezeiraud** [reply]
Thank you. Sorry for the issue.
